apps/iam/templates/iam/manage.html

```
@@ -0,0 +1,92 @@
+{% load i18n %}
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)